src/commands/migrateQiskitCode.ts

```diff
@@ -1,6 +1,10 @@
 import vscode from "vscode";
 import { migrateCode } from "../services/qiskitMigration";
 import { setDefaultStatus, setLoadingStatus } from "../statusBar/statusBar";
+import { getServiceApi } from "../services/common";
+import { DISMISS, FEEDBACK_RESPONSE_MSG, PROVIDE_FEEDBACK } from "../globals/consts";
+
+const MIGRATION_FEEDBACK_MSG = "Please share any detailed feedback you have about the Qiskit Code Assistant Migration. The details of your migration as well as this message will be sent to the Qiskit Code Assistant team in order to improve the service.";
```

Comment:
Why this is not part of `src/globals/consts.ts`?

Reply:
oversight... moved. thanks!